Tasks/AzureFileCopyV2/task.json

```diff
@@ -13,7 +13,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 2,
-        "Minor": 178,
+        "Minor": 179,
```

Comment:
Could you please bump the version in accordance with the current sprint as per [this guide](https://github.com/microsoft/azure-pipelines-tasks/blob/master/docs/taskversionbumping.md) for all tasks changed?

Reply:
done